### DISCUSSION TITLE
Meetings page is empty

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#676] [Q&A] [answered]
I deactivated the plugin then re-activated, and the meetings page is now blank. 
I see this has happened to another group here: 
https://wordpress.org/support/topic/meeting-page-disappeared/

Please advise. 
This is pretty urgent.

**Comment by kiyote33** [comment]
Hi @julien-paris,

This has happened to me in the past, and I have found simply restarting my server usually resolves the issue.  I believe I've also tried deleting the tsml plugin entirely and reloading from the WordPress directory with some success.  Let me know if either works for you. If not I'll search for another solution...

**Reply by julien-paris** [reply]
@kiyote33 thx for the answer. 
If I delete/re-install the plugin is there any loss of meeting data?

**Reply by kiyote33** [reply]
To be prudent, I would export the meetings to a csv file first. I just ran a test and it did drop the meeting records. I was running from a subset  of your data source (Munich) so it only took a second to reload those records.

**Reply by julien-paris** [reply]
Just talked to the host, and being on a shared server they can't restart. 
So to be clear, if I delete the plugin and reinstall it,  I will need to import the all meetings again? 
I also assume our external data sources will be lost. Is this correct?

**Reply by kiyote33** [reply]
I would assume the data sources might also be lost, so I would ensure I've got a list of the URL for backup.

Another suggestion might be to manually create a blank **Meetings** page with a **meetings** slug. I haven't been able to recreate the situation your having right now, so I can't test whether that works or not.

**Reply by julien-paris** [reply]
I have done the page, and added "under maintenance"

From the host: 
"I'd suggest asking why restarting the server might help, as that might give us some idea of what else we might be able to do to assist.
There don't seem to be any stuck processes on the site that might need to be stopped."

Please advise, thanks

**Reply by julien-paris** [reply]
Also, with 900+ meetings, how stable is the important process? 
I cannot afford this to go astray

**Reply by kiyote33** [reply]
Again, to be prudent you should export the meetings to a csv file first so you can get your meetings back quickly if need be.  I find the import (both csv and data sources) to be fairly quick and reliable.

Before you do the more drastic measures, I would suggest you exhaust the easiest things first.  Number one on my list would be to reboot your local machine before you do anything else.  That way you can be assured that you are working with a pristine environment.

**Reply by kiyote33** [reply]
Second on my list would be to do a simple restore from a backup. I'm assuming that you do regular backups of your website.

**Reply by julien-paris** [reply]
Sorry I don't understand. My local machine? My PC? Not the server? 
If you mean the server, as I mentioned: 
- the host cannot reboot it bc it's a shared hosting environment. 
- they asked why we needed to reboot, to see if the is any other issue they can help with.

**Reply by kiyote33** [reply]
Sorry if I'm not being clear.  I'm just trying to walk you through what I would try to resolve this issue. Sometimes activities in my local environment affect certain operations that I'm not aware of, so a reset to a pristine environment gives me a good clean starting place. If restarting the server is not an option for you then move on to the next suggestion. Restart your local PC.

**Reply by kiyote33** [reply]
Another thing I would try is to insert the **[tsml_ui]** shortcode on the text tab of the blank meetings page. That should work, but the interface is a bit different to what you are used to.

**Comment by julien-paris** [comment]
That worked indeed, but the single meeting page does not work correctly: 
https://alcoholics-anonymous.eu/meetings/?meeting=aa-beginners-welcome-eindhoven

- I have rebooted my PC
- I have a meetings CSV export
- I have site backups, both manual and auto

**Reply by kiyote33** [reply]
I'm guessing your saying the meeting detail page is not working correctly because the map doesn't display.  The tsml_ui interface is designed to work with mapbox. Are you using the Google API by chance?

**Reply by kiyote33** [reply]
See how you have site backups, you can always resort to your most recent which should put every thing back the way was before. That may be your best solution...

**Reply by julien-paris** [reply]
Yes indeed, that is why. 
I just tried 2 things
1. The 3 geocoding methods: 
Legacy/Google API/BETA API and get the same results
2. removing my google maps API Key. 
Nothing worked

**Reply by julien-paris** [reply]
Also just realized that when I delete the Google maps API key it reappears.

**Reply by julien-paris** [reply]
Ok this was solved by removing the Google Maps API and adding a mapbox API.

**Reply by kiyote33** [reply]
The tsml_ui interface is most probably the future for the 12 Step Meeting List plugin. We currently support both, but I've heard talk of removing support for the legacy interface and making tsml_ui the default.  So, your sort of on the leading edge with it.  You can of course go back to the legacy interface by simply removing the [tsml_ui] shortcode. Theoretically, it should reappear automatically.  Good luck!

**Reply by julien-paris** [reply]
Thanks. So for now things are displaying ok, I appreciate your help. 
Please keep in mind we still have an issue with 
https://github.com/code4recovery/12-step-meeting-list/discussions/663

**Reply by kiyote33** [reply]
Yep, still working it.  I'll update you with a note on that thread...

**Comment by julien-paris** [comment, marked as answer]
ISSUE RESOLVED: this came from a group which had the same problem: 
**Resolution: Resave permalinks**
Thanks, 
Julien

**Reply by kiyote33** [reply]
Makes sense. Thanks for passing it on...
